addons/account/static/src/js/account_reconciliation_widgets.js

```diff
@@ -1580,6 +1581,15 @@ var bankStatementReconciliation = abstractReconciliation.extend({
         else
             this.$(".show_more").hide();
     },
+
+    goToCreatePreset: function() {
+        this.action_manager.do_action({
+            type: 'ir.actions.act_window',
+            res_model: 'account.reconcile.model',
+            views: [[false, 'form']],
+            target: 'current'
```

Comment:
target: 'new' will left the current windows open, IMHO it is better than lose the current windows.